### PR TITLE
COMMERCE-4923 Fixed option reset on update

### PR DESCRIPTION
**Pull request by alessiorendina**

### CODE REVIEW

**Comment by Commando18** [comment]
ci:test:relevant

**Comment by Commando18** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-product-definitions-web/src/main/resources/META-INF/resources/edit_attachment_file_entry.jspf [hunk 1]
```diff
@@ -96,34 +96,32 @@
...
+%>
+
+<aui:script require="commerce-frontend-js/utilities/forms/formsHelper as FormsHelper">
 	var fieldValues = [];
 
+	try {
+		fieldValues = JSON.parse('<%= fieldValues %>');
+	}
+	catch(_ignore) {
+	}
```
Comment (dsitu) [inline modules/apps/commerce/commerce-product-definitions-web/src/main/resources/META-INF/resources/edit_attachment_file_entry.jspf:114]: @Commando18 running `ant format-source-current-branch` on portal-impl should auto fix this.
It should format it to something like this `catch (_ignore) {}`.

**Review by dsitu — CHANGES_REQUESTED** [review]

**Comment by Commando18** [comment]
ci:test:relevant

**Review by dsitu — APPROVED** [review]

**Comment by Commando18** [comment]
ci:test:relevant

**Comment by Commando18** [comment]
ci:test:sf

**Comment by marco-leo** [comment]
ci:test:relevant

**Comment by marco-leo** [comment]
ci:forwad

**Comment by marco-leo** [comment]
ci:forward

**Pull request closed without merging**